upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -643,6 +649,79 @@ func validateDNS(cluster *api.Cluster, cloud fi.Cloud) error {
 		glog.V(2).Infof("Found NS records for %q: %v", dnsName, hosts)
 	}
 
+	// TODO: Move to update
+	if os.Getenv("KOPS_FF_SKIP_DNS_PRECREATE") == "" {
```

Comment:
Good news ... I added a real feature flag implementation.  (Still pretty experimental)  But I'm going to update this PR to use that...